presto-iceberg/src/test/java/io/prestosql/plugin/iceberg/TestIcebergSmoke.java

```diff
@@ -73,8 +75,17 @@ protected QueryRunner createQueryRunner()
         return createIcebergQueryRunner(ImmutableMap.of());
     }
 
-    @Test
-    public void testShowCreateSchema()
+    @DataProvider(name = "allFormats")
```

Comment:
nit: I think `name = "allFormats"` is not needed if it matches method name.

Reply:
Removed.

Reply:
Nice, I didn't' know about that. Makes it cleaner.